keyboards/mechlovin/kanu/info.json

```
@@ -0,0 +1,87 @@
+{
+    "keyboard_name": "Kanu",
+    "url": "",
+    "maintainer": "Team Mechlovin'",
+    "width": 16,
+    "height": 5,
+    "layouts": {
+        "LAYOUT": {
```

Or whatever this layout maps to
```suggestion
        "LAYOUT_all": {
```